hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/OzoneManagerProtocol.java

```diff
@@ -216,6 +217,13 @@ OmKeyLocationInfo allocateBlock(OmKeyArgs args, long clientID,
    */
   void renameKey(OmKeyArgs args, String toKeyName) throws IOException;
 
+  /**
+   * Rename existing keys within a bucket.
+   * @param keyMap The key is new key name nad value is original key OmKeyArgs.
+   * @throws IOException
+   */
+  void renameKeys(Map<String, OmKeyArgs> keyMap) throws IOException;
```

Comment:
Should we make thie Map<OmKeyArgs, OmKeyArgs> so that the batch of the rename does not to have the same volume/bucket name, e.g. /vol1/bucket1/key1->/vol1/bucket2/key2 and /vol2/bucket1/key1->/vol2/bucket1/key2?

Reply:
I'll make this to Map<OmKeyArgs, String> to consistent with the renameKey() API.

In addition, the API was added to OzoneBucket and currently only supports renaming keys under the same bucket. 
`OzoneBucket bucket = volume.getBucket(bucketName);`
`Map<String, String> keyMap = new HashMap();`
`keyMap.put(keyName1, newKeyName1);`
`bucket.renameKeys(keyMap);`